packages/gestalt/src/TextArea.js

```diff
@@ -3,16 +3,19 @@
 import * as React from 'react';
```

Comment:
One interesting bug I found in Chrome is that they always add an extra 4px of spacing to the bottom of each `<textarea />` https://bugs.chromium.org/p/chromium/issues/detail?id=89530

Reply:
Not going to address that in this PR since it'll break in other browsers.